conda-forge.yml

```diff
@@ -4,3 +4,6 @@ bot:
 github:
   branch_name: main
   tooling_branch_name: main
+azure:
+  store_build_artifacts: True
```

Comment:
You can remove these two. BTW, when on  linux, you can easily test the build locally using our build script: `python build-locally.py 2>&1 | tee log`

Reply:
Yep - I've removed this again now.